README.md

```diff
@@ -85,10 +85,16 @@ For issues related to Windows Mixed Reality that aren't directly related to the
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). 
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-# Useful Resources on Microsoft Windows Dev Center
-| ![Academy](External/ReadMeImages/icon_academy.png) [Academy](https://developer.microsoft.com/en-us/windows/mixed-reality/academy)| ![Design](External/ReadMeImages/icon_design.png) [Design](https://developer.microsoft.com/en-us/windows/mixed-reality/design)| ![Development](External/ReadMeImages/icon_development.png) [Development](https://developer.microsoft.com/en-us/windows/mixed-reality/development)| ![Community)](External/ReadMeImages/icon_community.png) [Community](https://developer.microsoft.com/en-us/windows/mixed-reality/community)|
+# Useful resources on the Mixed Reality Dev Center
+| ![Discover](External/ReadMeImages/icon_academy.png) [Discover](https://docs.microsoft.com/en-us/windows/mixed-reality/)| ![Design](External/ReadMeImages/icon_design.png) [Design](https://docs.microsoft.com/en-us/windows/mixed-reality/design)| ![Develop](External/ReadMeImages/icon_development.png) [Develop](https://docs.microsoft.com/en-us/windows/mixed-reality/development)| ![Distribute)](External/ReadMeImages/icon_community.png) [Distribute](https://docs.microsoft.com/en-us/windows/mixed-reality/implementing-3d-app-launchers)|
 | :--------------------- | :----------------- | :------------------ | :------------------------ |
-| See code examples. Do a coding tutorial. Watch guest lectures.          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Join open source projects. Ask questions on forums. Attend events and meetups. |
+| Learn to build mixed reality experiences for HoloLens and immersive headsets (VR).          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Get your app ready for others and consider creating a 3D launcher. |
+
+# Useful resouces on Azure
+| ![Azure Spatial Anchors](External/ReadMeImages/icon_academy.png) [Azure Spatial Anchors](https://docs.microsoft.com/en-us/azure/spatial-anchors/)| ![Speech Services](External/ReadMeImages/icon_design.png) [Speech Services](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/)| ![Vision Services](External/ReadMeImages/icon_development.png) [Vision Services](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/)|       
```

Comment:
Do you know if it's possible to add a line break or something to the caption? it looks like this on chrome:

![image](https://user-images.githubusercontent.com/5840182/55902937-52d35080-5b81-11e9-9c70-2d3a284c410a.png)

Reply:
I shortened it to 'Spatial Anchors' - hopefully that fixes it.